app/views/devise/registrations/edit.html.haml

```diff
@@ -0,0 +1,23 @@
+%h2 Edit Profile #{resource_name.to_s.humanize}
+= simple_form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
+  .form-inputs
+    = f.input :given_name, required: true, autofocus: true
+    = f.input :last_name, required: true
+    = f.input :brith_date, required: true
+    = f.input :email, required: true
+
+    - if devise_mapping.confirmable? && resource.pending_reconfirmation?
+      %p Currently waiting confirmation for: #{resource.unconfirmed_email}
+
+    = f.input :password, autocomplete: 'off', hint: "leave it blank if you don't want to change it", required: false
+    = f.input :password_confirmation, required: false
+    = f.input :current_password, hint: 'we need your current password to confirm your changes', required: true
+
+  .form-actions
+    = f.button :submit, 'Update'
+
+%h3 Cancel my account
+%p Are you Unhappy?
```

Comment:
this will throw an error, if you have multiple lines, they should all be nested below `%p`